### ISSUE TITLE
Forward slashes are ignored when specifying string pattern

### ISSUE THREAD

**Issue by svile** [fastify/fluent-json-schema#121]
## 🐛 Bug Report

It seems that in attempts to auto escape the regex for json compatibility the library strips out (or ignores) escaped `/`

## To Reproduce

```ts
import S from 'fluent-json-schema';

const cronlike = S.string().pattern(
  /^((((\d+,)+\d+|(\d+(\/|-)\d+)|\d+|\*)\s?){5,6})$/,
);
console.log(cronlike.valueOf());

const cronlikeRegex = RegExp(
  /^((((\d+,)+\d+|(\d+(\/|-)\d+)|\d+|\*)\s?){5,6})$/,
);
console.log(cronlikeRegex);
```

## Expected behavior

Forward slashes should also be escaped.

```js
{
  '$schema': 'http://json-schema.org/draft-07/schema#',
  type: 'string',
  pattern: '^((((\\d+,)+\\d+|(\\d+(\\|-)\\d+)|\\d+|\\*)\\s?){5,6})$/'
}
/^((((\d+,)+\d+|(\d+(\/|-)\d+)|\d+|\*)\s?){5,6})$/
```

## Your Environment

- *node version*: v14.16.1
- *fluent-json-schema*: 2.0.4
- *os*: Mac

**Comment by aboutlo** [comment]
Thank you @svile to report this. 

Would you like to open a PR to fix it? 
You can add the test case here: https://github.com/fastify/fluent-json-schema/blob/master/src/StringSchema.test.js#L97

**Comment by svile** [comment]
Actually, looks like I'll have some spare time today, so I'll take it.

**Comment by climba03003** [comment]
I think the current behavior is correct. We should remove the starting forward slash and the ending forward slash when you are using pattern.

[IETF Reference](https://tools.ietf.org/id/draft-handrews-json-schema-validation-01.html#rfc.section.6.3.3)
[JSON Schema](https://json-schema.org/understanding-json-schema/reference/regular_expressions.html#example)

**Comment by aboutlo** [comment]
resolved by #122
